.github/workflows/release.yml

```diff
@@ -36,4 +36,4 @@ jobs:
 
       - name: Sync UI Bundle to S3
         run: |
-          aws s3 sync --follow-symlinks --include '*.zip' --no-progress build s3://${{ secrets.STATIC_CONTENT_BUCKET }}/build
+          aws s3 sync --follow-symlinks --include '*.zip' --no-progress build s3://${{ secrets.STATIC_CONTENT_BUCKET }}/build
```

Comment:
Remove changes to both workflow files from this PR, as they don't belong. It's good hygiene and will help any future troubleshooting involving those files.